experiences/homepage/homepage.test.js

```diff
@@ -47,6 +47,66 @@ const homepageTest = function() {
 		expect(cardExists).equal(true);
 	});
 
+	it('when scrolling a few pixels down, the card pops up the same amount of pixels', async() => {
+		const cardExists = await isCardExisting(
+			browser,
+			config.cards.homepage.cardSelector
+		);
+
+		expect(cardExists).equal(true);
+
+		//Make the card appear a little bit
+		await scrollBy(browser, 100);
+
+		// Test scroll bounce
+		let isAtExpectedSnapPoint = await isAtSnapPoint(
+			browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.minimised
+		);
+
+		expect(isAtExpectedSnapPoint).equal(true);
+
+		await scrollBy(browser, 20);
+
+		isAtExpectedSnapPoint = await isAtSnapPoint(
+			browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.minimised - 0.1
+		);
+		expect(isAtExpectedSnapPoint).equal(true);
+
+		await scrollBy(browser, 20);
+
+		isAtExpectedSnapPoint = await isAtSnapPoint(
+			browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.minimised - 0.2
```

Comment:
I did not wrapped this inside a loop due to the problem with loops and async/await statements. Any idea on how to write with less code repetition?

Reply:
`await arrayOfScrollsToDo.reduce((prev, value)=>{
		return prev.then(()=>{
			scrollBy(browser, value);
			checkSnappoint();
			return new Promise(resolve => setTimeout(resolve, 100));
		});
	}, Promise.resolve());`
	
	Something like this

Reply:
```
await arrayOfScrollsToDo.reduce((prev, value)=>{
		return prev.then(async()=>{
			scrollBy(browser, value);
			return await checkSnappoint();
		});
	}, Promise.resolve());
```

Reply:
something like this could work, chaining the checks